content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -142,36 +285,40 @@ resources:
       - aescbc:
           keys:
             - name: key1
+              # See the following text for more details about the secret value
               secret: <BASE 64 ENCODED SECRET>
-      - identity: {}
+      - identity: {} # this fallback allows reading unencrypted secrets;
+                     # for example, during initial migratoin
 ```
 
-To create a new Secret, perform the following steps:
-
-1. Generate a 32-byte random key and base64 encode it. If you're on Linux or macOS, run the following command:
-
-   ```shell
-   head -c 32 /dev/urandom | base64
-   ```
+Place that base64 encoded value in the `secret` field of the encryption configuration file.
 
-1. Place that value in the `secret` field of the `EncryptionConfiguration` struct.
-1. Set the `--encryption-provider-config` flag on the `kube-apiserver` to point to
-   the location of the config file.
+### Update the configuration of each API server {#api-server-config-update}
 
-   You will need to mount the new encryption config file to the `kube-apiserver` static pod. Here is an example on how to do that:
+You will need to mount the new encryption configuration file to the `kube-apiserver`
```

Comment:
nit: Could shorten to,
Mount the new encryption configuration ...

Reply:
I'll leave this as I'm not rewording here.